shims/common/src/main/scala/org/apache/gluten/config/GlutenConfig.scala

```diff
@@ -509,6 +510,9 @@ object GlutenConfig {
       (SQLConf.CASE_SENSITIVE.key, SQLConf.CASE_SENSITIVE.defaultValueString),
       (SQLConf.IGNORE_MISSING_FILES.key, SQLConf.IGNORE_MISSING_FILES.defaultValueString),
       (SQLConf.LEGACY_TIME_PARSER_POLICY.key, SQLConf.LEGACY_TIME_PARSER_POLICY.defaultValueString),
```

Comment:
I don't know why LEGACY_TIME_PARSER_POLICY is placed here, looks like it always sets the default value to backend.